src/api/telegram-subscription/content-types/telegram-subscription/schema.json

```diff
@@ -16,17 +17,23 @@
       "required": true,
       "unique": false
     },
-    "login": {
-      "type": "string",
-      "required": true
+    "auth_date": {
```

Comment:
why do we mix camel-case and snakecase?

Reply:
Initially I left snake_case to be consistent with data format from Telegram API.
But we map the data anyway, changed to camelCase on our side